source/PlayState.hx

```diff
@@ -1836,11 +1839,7 @@ class PlayState extends MusicBeatState
 
 		super.update(elapsed); //TEST
 
-		if(ratingString == '?') {
-			scoreTxt.text = 'Score: ' + songScore + ' | Misses: ' + songMisses + ' | Rating: ' + ratingString;
-		} else {
-			scoreTxt.text = 'Score: ' + songScore + ' | Misses: ' + songMisses + ' | Rating: ' + ratingString + ' (' + Math.floor(ratingPercent * 100) + '%)';
-		}
+		scoreTxt.text = 'Score: ' + songScore + ' | Misses: ' + songMisses + ' | Rating: ' + ratingString + ((ratingString == '?') ? '' : ' (' + Math.floor(ratingPercent * 100) + '%)');
```

Comment:
Bit of an excessive ternary, don't you think?

Reply:
Readability > one-liners IMO.

Reply:
totally understood, personally I just like my streamlined code lol 

a better solution were to have a more readable string outside that's edited, then add that to scoreTxt.text, because that's the best of both worlds I think

Reply:
I feel 'ya, I use a lot of one-liner stuff for personal projects, but I try to lessen them when I'm making stuff I want people to contribute to. :p
Having some sort of outside string (or a function with parameters you'd pass in) may be better (and promotes arguably less hardcode too).